spec.md

```diff
@@ -509,6 +624,15 @@ To delete a manifest, perform a `DELETE` request to a path in the following form
 Upon success, the registry MUST respond with a `202 Accepted` code.
 If the repository does not exist, the response MUST return `404 Not Found`.
 
+When deleting an Image or Artifact manifest that contains a `refers` field, and the [Referrers API](#listing-referrers) returns a 404, clients SHOULD:
+
+1. Pull the Image Index using the [referrers tag schema](#referrers-tag-schema).
+1. Delete the descriptor from the manifest list to the deleted manifest.
```

Comment:
Minor, but I personally found the second step language a bit difficult to parse; maybe because "manifest list" is a known image type from the Docker v2 spec. Once I understood that **list** refers to the index's array of manifests it made more sense, but here's a potential reword (that also gets rid of the word "delete" since the verb "delete" isn't the most natural word to use with array operations, IMO):

*Remove the descriptor entry from the array of manifests that references the deleted manifest.*

Reply:
Makes sense, PR updated. Thanks!